sdk/formrecognizer/azure-ai-formrecognizer/azure/ai/formrecognizer/_form_base_client.py

```diff
@@ -64,6 +64,11 @@ def __init__(self, endpoint, credential, **kwargs):
             }
         )
 
+        try:
+            endpoint = endpoint.rstrip("/")
+        except AttributeError:
+            raise ValueError("Parameter 'endpoint' must be a string.")
+
```

Comment:
we'll want for async too (realize you might not have started converting async yet, but wanted to call out)